docs/getting-started/first-steps.md

```diff
@@ -0,0 +1,7 @@
+# First Steps
+
+TODO: How to get help
```

Comment:
Some suggested wording:

`rig` often offers two levels of commands which allows for grouping all actions related to a similar concept together. This is most evident with the commands related to a project. To see a list of the available commands run `rig help`. To see subcommands that are available for a given command run `rig help [command]`. For example, in a project directory, running `rig help project` will show you a list of the project specific actions available to you.